db.ts

```diff
@@ -72,4 +72,20 @@ const createNewHighScore = async (user: User, typeOfQuiz: string, newHighScore:
     }
 }
 
-export { connect, createUser, getUser, createNewHighScore }
\ No newline at end of file
+const deleteBlacklist = async (user: User, quoteId: string) => {
+    try {
+        await client.db("TheOneQuiz").collection("Users").updateOne({ username: user.username }, { $pull: { blacklist: { quoteId: quoteId } } });
+    } catch (err) {
+        console.log(err);
+    }
+}
+
+const editBlacklist = async (user: User, quoteId: string, newComment: string) => {
+    try {
+        await client.db("TheOneQuiz").collection("Users").updateOne({ username: user.username, 'blacklist.quoteId': quoteId }, { $set: { 'blacklist.$.comment': newComment }});
```

Comment:
You made a tiny mistake here, but it causes the function to not work. You have to use "quote_id" as a parameter for "blacklist.quote_id", not "blacklist.quoteId".

So it should be:
`        await client.db("TheOneQuiz").collection("Users").updateOne({ username: user.username, 'blacklist.quote_id': quoteId }, { $set: { 'blacklist.$.comment': newComment }});
`